analytics/core.go

```diff
@@ -0,0 +1,52 @@
+package analytics
+
+import (
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/config"
+)
+
+type PBSAnalyticsModule interface {
```

Comment:
I think this interface, and all its methods, need to be doc'd. Some conventions are here: https://blog.golang.org/godoc-documenting-go-code

Interface contracts are tricky... but _essential_ if you want things to work smoothly. They need to be specific enough that the callers (e.g. the `exchange`) know exactly when they should use each function, but general enough that none of the possible legitimate implementations contradict them in any way.

It's a balancing act. Take a stab at it, and we'll see how it goes.